tests/template/hdf5_util_test.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""Test for :mod:`sirepo.template.hdf5_util`
+
+:copyright: Copyright (c) 2023 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import os
+import shutil
+import subprocess
+import time
+
+
+def test_read_while_writing():
+    def _read_not_written(case_work_dir):
+        p = _proc(case_work_dir, "read.py", "does_not_exist_at_first.h5")
+        time.sleep(10)
```

Comment:
See if the sleeps can be shorted a bit - this will slow up the builds otherwise.